tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsTests.NetFwk.cs

```diff
@@ -288,7 +288,7 @@ private async Task RunClientCredsAsync(Cloud cloud, CredentialType credentialTyp
             var appCacheRecorder = confidentialApp.AppTokenCache.RecordAccess();
             Guid correlationId = Guid.NewGuid();
             authResult = await confidentialApp
-                .AcquireTokenForClient(settings.AppScopes)
+                .AcquireTokenForClient(settings.AppScopes)                
```

Comment:
delete this new whitespace that you added?

Reply:
removed.